.gitignore

```diff
@@ -49,7 +49,8 @@ coverage.all
 
 /bin
 /dist
-/custom
+!/custom/conf/app.ini
```

Comment:
The file is a user customerize one, I don't think it's right.

Reply:
Yeah, working on it

Reply:
Fixed